.npmignore

```diff
@@ -0,0 +1,26 @@
+# https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package
+
+# build tool config
+Jenkinsfile
+.travis.yml
+
+# code examples
+examples
+
+# docs
+CONTRIBUTING.md
+
```

Comment:
Fair enough, we'll leave it in. Perhaps it was/will be included in the npm generated pages at somepoint.